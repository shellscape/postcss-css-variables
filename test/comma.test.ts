import { fixture, process } from './helpers';

const tests = {
  'work use the correct variable in comma separated selector': 'comma-separated-variable-usage',
  'work with variables defined in comma separated selector': 'comma-separated-variable-declaration'
};

describe('comma', async () => {
  test.each(Object.entries(tests))('%s', async (_desc, file) => {
    const css = await fixture(`comma/${file}`);
    const result = await process(css);
    expect(result).toMatchSnapshot();
  });
});
