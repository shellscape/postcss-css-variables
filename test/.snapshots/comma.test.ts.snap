// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`comma > work use the correct variable in comma separated selector 1`] = `".some-foo-namespace .foo{color:red}.some-bar-namespace .bar{color:blue}"`;

exports[`comma > work with variables defined in comma separated selector 1`] = `".some-bar-namespace * ~ *{margin-top:13px}"`;
